catalogue/webapp/utils/iiif.ts

```diff
@@ -290,19 +291,14 @@ export function getVideo(
   );
 }
 
-export function getAudio(
-  iiifManifest: IIIFManifest
-): IIIFMediaElement | undefined {
-  const videoSequence =
-    iiifManifest &&
-    iiifManifest.mediaSequences &&
-    iiifManifest.mediaSequences.find(sequence =>
+export function getAudio(iiifManifest: IIIFManifest): IIIFMediaElement[] {
+  const audioSequences = (iiifManifest.mediaSequences || [])
+    .flatMap(sequence =>
```

Comment:
snazzy! do we need to add it to the polyfill.io list?

Reply:
ah good question – I thought it was there. If not, then almost certainly, yes